.github/workflows/master.yml

```diff
@@ -394,15 +394,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      # from https://github.com/marketplace/actions/setup-minikube-kubernetes-cluster
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.7.2
-        with:
-          minikube version: 'v1.28.0'
-          kubernetes version: 'v1.25.4'
-          github token: ${{ secrets.GITHUB_TOKEN }}
-          driver: docker
-          start args: '--extra-config=kubeadm.ignore-preflight-errors=NumCPU --force --cpus 2 --memory 4096'
+        run: |
+          # https://minikube.sigs.k8s.io/docs/start/
+          curl -LO https://github.com/kubernetes/minikube/releases/download/v1.29.0/minikube-linux-amd64
```

Comment:
let's pullout `KUBERNETES_VERSION` and `MINIKUBE_VERSION` to L36

Reply:
Good point!